components/server/src/workspace/workspace-service.ts

```diff
@@ -568,13 +568,10 @@ export class WorkspaceService {
                 daysToLive = daysToLive * 2;
             }
             deletionEligibilityTime.setDate(deletionEligibilityTime.getDate() + daysToLive);
-            if (
-                workspace.deletionEligibilityTime &&
-                workspace.deletionEligibilityTime > deletionEligibilityTime.toISOString()
-            ) {
+            if (new Date() > deletionEligibilityTime) {
```

Comment:
```suggestion
            if (new Date().toISOString() > deletionEligibilityTime.toISOString()) {
```

Reply:
`.getTime()` would also be fine with me. But the unqualified use of operators on objects makes me shiver :woozy_face:

Reply:
Learned about [`new Date()[Symbol.toPrimitive]("number")`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Symbol.toPrimitive) today because of this, thanks!

Let's use 'em ISOs

Reply:
These are the things that make you question your life choices when still working with Javascript in 2025 :stuck_out_tongue_closed_eyes: :rofl: